index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" 
+    content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/index.css" >
+    <title>Portfolio</title>
+</head>
+<body>
+  <header id="header">
+    <h1>My Logo</h1>  
+    
+    <button id="menu-btn"></button>
+
+    <nav id="header-nav">
+      <ul>
+        <li><a href="">Portfolio</a></li>
+        <li><a href="">About</a></li>
+        <li><a href="">Contact</a></li>
+        <li><a href="">Mail</a></li>
+      </ul>
+    </nav> 
+
+  </header>
```

Comment:
You did a good job working on your toolbar section! 🤓 However, there's still some room to improve:


|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/235172050-aa7c27fc-81d5-4f9d-ab97-6642337d9824.png) | ![image](https://user-images.githubusercontent.com/77069011/235171797-2e731253-5080-46cb-be5b-2035500058c2.png) |

- [ ] 🚧 Please consider reducing the blank space at the top of the toolbar! 🔧 
- [ ] 🚧 By convention, the logo of your page should be a link to the home page, please consider using an anchor `<a>` tag instead of an `<h1>` to wrap it! This will improve tremendously the user experience. 🔧 
- [ ] 🚧 Please consider downloading the font, as currently none of the fonts you specified are getting applied. 🔧
- [ ] 🚧 Please include the hamburger button inside the `<nav>` tag, as it is part of the navigation of your page! 🔧  

One of the main goals of this project is that you demonstrate the ability to follow as closely as possible a given design! Please consider applying those changes. 🤖

Reply:
What a "hamburger button" is in my "nav" tag !?